src/utils/useSSO.ts

```diff
@@ -23,6 +24,22 @@ export const useSSO = (authServiceUrl: string) => {
     if (event.origin === authServiceUrl && event.data.source === "zesty") {
       setMessage(event.data);
       if (event.data.status === "200") {
+        // Set the cookie for .content.one domain as well
+        if (event.data.token) {
```

Comment:
I'm not sure I understand what the purpose of this change is. The prior logic did not deal with setting a cookie. Why does it now? How does using a cookie fit into the SSO process?